src/legacy/status_im/ui/screens/stickers/views.cljs

```diff
@@ -38,7 +38,7 @@
                              (or owned free)
                              (re-frame/dispatch [:stickers/install-pack id])
                              not-enough-snt? nil
-                             :else (re-frame/dispatch [:stickers/buy-pack id]))}
+                             :else (fn [] (js/alert "feature no longer supported")))}
```

Comment:
Is this okay to do? We have sticker support in the new UI for MVP afaik. 
right @Parveshdhull ?
cc @cammellos

Reply:
Hi @J-Son89, Thank you for pinging.
We do have stickers in the design file but they are not implemented yet.
I don't know if we will use this code or not, but maybe for now just comment out for future reference. wdyt?

(Means event part, ui is already inaccessible)

Reply:
Thanks for the help @Parveshdhull! 🙏 
Ok perfect, in that case this change is "safe" so. I will prefer to remove the code instead actually as we have git history for that very purpose and removing stale/commented out code leaves the codebase feeling cleaner and more maintained. However I will remove stickers in a follow up as I don't want this pr to grow in scope.
 
On a further point I think we should remove as much of the "legacy" folder as being unused as often when we are making "new" ui integrations there is some mixing of the legacy and new and then it becomes messy to maintain.  I'll try make a push to remove more this month.